plugins/algoltek-aux/fu-algoltek-aux.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2024 Algoltek <Algoltek, Inc.>
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+#[repr(u8)]
+enum FuAlgoltekAuxCmd {
+    Rdr = 0x06,
+    Wrr,
+    Rdv,
+    En,
+    Wrf = 0x10,
+    Isp = 0x13,
+    Ers = 0x19,
+    Bot = 0x1D,
+    Rst = 0x20,
+    Vid = 0x22,
+}
+
+#[derive(ParseStream, ValidateStream)]
+struct FuStructAlgoltekAuxProductIdentity {
+    header_len: u8,
```

Comment:
@YiJaneWu is this field always the size of the struct, i.e. `0x4B`?

Reply:
Yes, it's always the size of the struct.

Reply:
@YiJaneWu are you sure? :)

	11:01:43.519 FuStruct             FuStructAlgoltekAuxProductIdentity:
	  header_len: 0x8
	  product_name_len: 0x10
	  product_name: AG9421-TEST
	  version_len: 0x30
	  version: AG9421_00.09.19_3.1.4_4.2.0-LVFS_AUX_v10-2

This is with the `firmware.bin` you sent me...

Reply:
Ohh, it's the length of `header`, not the length of the `FuStructAlgoltekAuxProductIdentity`!

Reply:
Sorry! I just didn’t understand what you meant. 
Yes! It represents the length of header !(ALGOLTEK)